src/api.ts

```diff
@@ -11,6 +11,6 @@ function wait(delay: number) {
 export async function getPeople(): Promise<Person[]> {
   // keep this delay for testing purpose
   return wait(500)
-    .then(() => fetch(API_URL))
+    .then(() => fetch(`${API_URL}`))
```

Comment:
Using template literals for a static string like `${API_URL}` is unnecessary. You can directly use `API_URL` without the template literals.